app/viewmodels/package.scala

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Confusing filename, maybe move Heading, Link and TaxYears to TaxCalculationViewModel as it's only used there and the rest to ViewUtils